web/source/text/processor.ts

```diff
@@ -675,22 +677,22 @@ namespace com.keyman.text {
     getVKDictionaryCode(keyName: string) {
       let keyman = com.keyman.singleton;
       var activeKeyboard = this.activeKeyboard;
-      if(!activeKeyboard['VKDictionary']) {
+      if(!activeKeyboard.scriptObject['VKDictionary']) {
```

Comment:
Notes for the future:

This function does some neat stuff.  The `if`-`then` block here should actually be part of the wrapper's `constructor`, now that it exists, as it creates the `VKDictionary` property on the script object.

Also, we'll be wanting something similar for headless stores, but that comes later.